linux_os/guide/system/permissions/partitions/mount_option_nosuid_removable_partitions/rule.yml

```diff
@@ -17,7 +17,11 @@ rationale: |-
     users to introduce SUID or SGID binaries from partitions mounted off of
     removable media would allow them to introduce their own highly-privileged programs.
 
-severity: unknown
+severity: {{% if product == "ol7" -%}}
+medium
+{{%- else -%}}
+unknown
+{{%- endif %}}
```

Comment:
Always replace `unknown`.

Reply:
@redhatrises acked, I'll update PR. 

Does it make sense to mention it in the Developer Guide?
As not being the rule contributor, change it's default severity value to one from prodtype specific security guide just looks not obvious.